build.gradle

```diff
@@ -68,7 +73,7 @@ ext {
 	springBootVersion = '2.5.5' // docs module
 	springDataVersion = '2021.1.0-RC1'
```

Comment:
This one has to be `2022.1.0-SNAPSHOT`.
And Boot I guess has to move to `2.6.0`.
Jackson `2.13.0`.
Kotlin `1.5.31`